miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -157,6 +152,14 @@ end
 #! - index is the index of the item to get.
 #! - VALUE is the value of the item.
 export.get_account_item
+    # authenticate that the procedure invocation originates from the account context
+    exec.authenticate_account_origin
+    # => [storage_offset, index, 0, 0, 0]
+
+    # apply offset to storage slot index
+    exec.account::offset_storage_index
+    # => [offset_index, 0, 0, 0]
```

Comment:
minor nit: here an other similar places I would use `index_with_offset` to make it less ambiguous.